### PR TITLE
add support for double arrow line operator

### PR DESCRIPTION
**Pull request by srkondam**
modified longleftrightarrow command to support line

### CODE REVIEW

File: src/commands/math/advancedSymbols.js [hunk 1]
```diff
@@ -160,7 +160,8 @@ LatexCmds.longleftarrow = bind(VanillaSymbol, '\\longleftarrow ', '&#8592;', 'le
 LatexCmds.longrightarrow = bind(VanillaSymbol, '\\longrightarrow ', '&#8594;', 'right arrow');
 LatexCmds.Longleftarrow = bind(VanillaSymbol, '\\Longleftarrow ', '&#8656;', 'left arrow');
 LatexCmds.Longrightarrow = bind(VanillaSymbol, '\\Longrightarrow ', '&#8658;', 'right arrow');
-LatexCmds.longleftrightarrow = bind(VanillaSymbol, '\\longleftrightarrow ', '&#8596;', 'left and right arrow');
+// update to Style command instead of VanillaSymbol
+//LatexCmds.longleftrightarrow = bind(VanillaSymbol, '\\longleftrightarrow ', '&#8596;', 'left and right arrow');
```
Comment (eluberoff) [inline src/commands/math/advancedSymbols.js]: I don't think this is right. Mathquill should stay synced up with the LaTeX spec. longleftrightarrow is the left right arrow character inline. I think you should restore this line.

File: src/commands/math/commands.js [hunk 2]
```diff
@@ -81,6 +81,7 @@ LatexCmds.underline = bind(Style, '\\underline', 'span', 'class="mq-non-leaf mq-
 LatexCmds.overline = LatexCmds.bar = bind(Style, '\\overline', 'span', 'class="mq-non-leaf mq-overline"');
 LatexCmds.overrightarrow = bind(Style, '\\overrightarrow', 'span', 'class="mq-non-leaf mq-overarrow mq-arrow-right"');
 LatexCmds.overleftarrow = bind(Style, '\\overleftarrow', 'span', 'class="mq-non-leaf mq-overarrow mq-arrow-left"');
+LatexCmds.longleftrightarrow = bind(Style, '\\longleftrightarrow ', 'span', 'class="mq-non-leaf mq-overarrow-leftright"', 'left and right arrow'); //&#8596;
```
Comment (eluberoff) [inline src/commands/math/commands.js]: Are you looking for the overleftrightarrow LaTeX command here?

Comment (srkondam) [reply src/commands/math/commands.js]: Yes, I am looking for overleftrightarrow.

Comment (srkondam) [reply src/commands/math/commands.js]: Eli, where can I find collection of all legal LaTeX commands? When I google I get different suggestions. I did look for overleftrightarrow.

Comment (eluberoff) [reply src/commands/math/commands.js]: Honestly, it depends on the package and is kind of a mess -- we end up resorting to googling a fair amount as well. Here I just figured that it should exist based on the previous two lines, and then confirmed it.

We often use LaTeXiT, which works on mac, to confirm how a character should appear. I'll push a fix to this commit (will amend your commit for a clean git history)

**Comment by srkondam** [comment]
overleftrightarrow looks good.Please merge the changes into default.

**Pull request merged**